sylvia-derive/src/multitest.rs

```diff
@@ -178,18 +166,19 @@ impl<'a> ContractMtHelpers<'a> {
             pub mod multitest_utils {
                 use super::*;
                 use #sylvia ::cw_multi_test::Executor;
-                use #sylvia ::derivative::Derivative;
-
-                #[derive(Derivative)]
-                #[derivative(Debug)]
-                pub struct #proxy_name <'app, MtApp, #(#generic_params,)* > {
-                    pub contract_addr: #sylvia ::cw_std::Addr,
-                    #[derivative(Debug="ignore")]
-                    pub app: &'app #sylvia ::multitest::App<MtApp>,
-                    _phantom: std::marker::PhantomData<( #(#generic_params,)* )>,
+
+                pub trait ContractImpl<'app, BankT, ApiT, StorageT, CustomT, WasmT, StakingT, DistrT, IbcT, GovT, #(#generic_params,)* >
```

Comment:
I think that it would be better to actually be consistent here with the multitest code generated for traits i.e. to use the name of the contract with suffix `Proxy` here instead of `ContractImpl`. For e.g. if the contract name is `SomeContract` then it would result in `SomeContractProxy` trait